<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de Visión Artificial - Entrega 05  Visión Estéreo</string>
  </property>
  <widget class="QFrame" name="imageFrameS">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameD">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Capture</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="colorButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Image</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>30</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>            Disparity</string>
   </property>
   <widget class="QPushButton" name="InitializeDisparity_Buttom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize Disparity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadGroundTruth_Buttom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Ground Truth</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="showCorners">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>180</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Corners</string>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameII">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameID">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadImages">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Images</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>620</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>José Antonio Pérez   y   Juan Antonio Silva     -    Entrega 05   (Visión Artificial  2019/20)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showCorners_2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>210</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Corner matching</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCDEstimated">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCTrue">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>370</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Estimated Disparity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>370</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>True Disparity</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progreso">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>300</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
